src/modules/localization/domain/kalman_filter/Dynamics.cpp

```diff
@@ -0,0 +1,40 @@
+#include "Dynamics.h"
```

Comment:
Use .hpp extension as the code is in c++. C++ does not care about extensions but let us keep things as streamlined as possible.